VirtoCommerce.CustomerReviews.Data/Services/CustomerReviewSearchService.cs

```diff
@@ -49,10 +49,12 @@ public async Task<GenericSearchResult<CustomerReview>> SearchCustomerReviewsAsyn
                     query = query.Where(x => criteria.ProductIds.Contains(x.ProductId));
                 }
 
-                if (criteria.ReviewStatus.HasValue)
+                if (!criteria.ReviewStatus.IsNullOrEmpty())
                 {
-                    var convertedStatus = (byte)criteria.ReviewStatus.Value;
-                    query = query.Where(x => x.ReviewStatus == convertedStatus);
+                    foreach(var statusId in criteria.ReviewStatus)
+                    {
+                        query = query.Where(r => r.ReviewStatus == (byte)statusId);
```

Comment:
will this allow to search by multiple statuses?

Reply:
Thanks for your review! Of course, this code doesn't work with multiple statuses. The field reviewStatus in entity model was byte, and when I casted reviewStatuses collection of int to byte and used in  "Contains()..." it didn't work. When I used reviewStatuses collection of int the result sql code contained cast from int to byte and it prevented using index. I have changed the type of field ReviewStatus to int and fixed the highlighted code.